generators/server/templates/src/main/docker/grafana/provisioning/datasources/datasource.yml.ejs

```diff
@@ -35,6 +35,7 @@ datasources:
     # <int> org id. will default to orgId 1 if not specified
     orgId: 1
     # <string> url
+    # Change localhost by host.docker.internal on MacOS
```

Comment:
Please change this comment to be clearer: `On Mac OS, replace `localhost` by `host.docker.internal`